File: OwinDemo/Models/ProductOwner.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace OwinDemo.Models
{
    public class ProductOwner 
    {
        public string CompanyName { get; set; }
        [DataType(DataType.MultilineText)]
        public string Description { get; set; }
        public int FoundedYear { get; set; }
        [DataType(DataType.Url)]
        public string WebsiteUrl { get; set; }
     }
}

File: OwinDemo/app/Model/productOwner.ts
export class Owners {
    public Id: string
    public UserName :string
    public ProductOwner: ProductOwner
    public isPending: boolean
     
}
export class ProductOwner {
    CompanyName: string
    FoundedYear: number
    sDescription: string
    WebsiteUrl: string   
}
